package.json

```diff
@@ -2,7 +2,8 @@
   "name": "xhd-wallet-api",
   "version": "0.0.1",
   "description": "A Typescript implementation of BIP-32 Ed25519 for Algorand ARC-52.",
-  "main": "dist/index.js",
+  "exports": "./dist/index.js",
```

Comment:
This key should be an object with types and default specified like the following:

```json
{
  "exports": {
      ".": {
        "types": "./dist/index.d.ts",
        "default": "./dist/index.js"
    },
  }
}
```